src/less/reset.less

```diff
@@ -19,3 +19,11 @@ button.rbc-input::-moz-focus-inner {
   border: 0;
   padding: 0;
 }
+
+button.rbc-event {
```

Comment:
its fine to move this up to the event.less file and combine the selector

Reply:
Under the same `.rbc-event` rule of `event.less`?